packages/arb-validator-core/gobridge/arbFactory.go

```diff
@@ -9,14 +9,13 @@ import (
 type arbFactory struct {
 	rollupContractAddress common.Address
 	client                *GoArbAuthClient
-	rollups               map[common.Address]*rollupData // contract instance to rollupData
 }
 
 func deployRollupFactory(m *goEthdata) {
+	m.rollups = make(map[common.Address]*arbRollup)
 	m.arbFactoryContract = &arbFactory{
 		rollupContractAddress: m.getNextAddress(),
 		client:                nil,
-		rollups:               make(map[common.Address]*rollupData),
 	}
 }
```

Comment:
The deploy functions would be better off as part of `goEthdata`. The way it's structured here, there's no relationship between `deployRollupFactory` and the rest of `arbFactory.go`

Reply:
fixed